### PR TITLE
Completed - html-web-page / Young Jae KIM

### PR DESCRIPTION
**Pull request by abodStorage**
html-webpage

### CODE REVIEW

File: 0200    html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <form>
+    <!-- Input: text - name -->
+    <input type="text" name="name" id="name" placeholder="Your Name"/>
+
+    <!-- Input: dropdown - tier plan selection -->
+    <select name="jelly_tier_plan" id="jelly_tier_plan">
+      <option value="default">Select Your Plan:</option>
+      <option value="basic">Basic Plan</option>
+      <option value="default">Deluxe Plan</option>
+      <option value="default">Jelly Giant Plan</option>
```
Comment (kurtyazdizadeh) [inline 0200    html-web-page/index.html]: these values should be unique and descriptive to the plan the customer is signing up for.

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Just a small change for setting the right `value` on your form for the dropdown.

File: 0200    html-web-page/index.html (hunk 1 above)
Comment (abodStorage) [reply 0200    html-web-page/index.html]: Updated code.

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**